src/app/member/loading.tsx

```diff
@@ -0,0 +1,52 @@
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from "@/components/ui/table";
+import { Skeleton } from "@/components/ui/skeleton";
+
+export default function Loading() {
```

Comment:
@muhammad-bilal-ali-khokhar I think we can create one global loading state and let's not make it look like Table.